src/pages/api/update-manual-review.js

```diff
@@ -49,38 +49,41 @@ const clearRemark = async (id) => {
 
 //function for manual review
 const updateManualRepo = async (req, res) => {
-  const repoId = req.query.id;
+  let repoIds = req.query.ids.split(",");
```

Comment:
Please explain why "let" was chosen here.

Reply:
need to modify value